### PR TITLE
Numerous admin-related, model changes

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

File: aldryn_jobs/cms_appconfig.py [hunk 1]
```diff
@@ -32,3 +32,7 @@ class JobsConfig(AppHookConfig):
 
     placeholder_jobs_list_bottom = PlaceholderField(
         'jobs_list_bottom', related_name='aldryn_jobs_list_bottom')
+
+    class Meta:
+        verbose_name = 'Aldryn Jobs configuration'
```
Comment (czpython) [inline aldryn_jobs/cms_appconfig.py]: no translations?

File: aldryn_jobs/models.py [hunk 2]
```diff
@@ -326,6 +327,7 @@ class Meta:
 class JobListPlugin(BaseJobsPlugin):
     """ Store job list for JobListPlugin. """
     jobopenings = SortedManyToManyField(JobOpening, blank=True, null=True,
+        verbose_name=_('job openings'),
         help_text=_("Select Job Openings to show or don't select any to show "
                     "last job openings. Note that Job Openings form different "
```
Comment (evildmp) [inline aldryn_jobs/models.py]: Spelling and syntax a bit odd here

**Comment by cyberstar** [comment]
LGTM :+1:

File: aldryn_jobs/models.py (hunk 2 above)
Comment (evildmp) [inline aldryn_jobs/models.py]: Suggest: "Select Job Openings to show. If none are selected, the most recent Job Openings from the selected app-config will be shown."

**Pull request merged**